qemu/tests/balloon_check.py

```diff
@@ -82,20 +84,21 @@ def memory_check(self, step, ballooned_mem):
             gcompare_threshold = max(gcompare_threshold,
                                      float(guest_mem_ratio) * self.ori_mem)
         # for windows illegal test:set windows guest balloon in (1,100),free
-        # memory of the OS shoul be as small as possible.
+        # memory of the OS should be as small as possible.
         if ballooned_mem >= self.ori_mem - 100:
```

Comment:
Shouldn't replace ori_mem with pre_mem here too?

Reply:
no need here, it's only for illegal test.